### PR TITLE
LPS-195690 [MARKETPLACE] Customer - License Selection for DXP Apps - Trial

### PR DESCRIPTION
**Pull request by juliocezar96**
[LPS-196887](https://liferay.atlassian.net/browse/LPS-196887)

### CODE REVIEW

**Comment by juliocezar96** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html><details><summary>More details about: <b>LPS-195690 Customer - License Selection for DXP Apps - Trial (2c)</b>.</summary>
              <br />
              <table>
              <tr>
            <th>Title</th>
            <td><a target="_blank" href="https://issues.liferay.com/browse/LPS-195690">Customer - License Selection for DXP Apps - Trial (2c)</a></td>
        </tr>
              <tr>
            <th>Type</th>
            <td>Story</td>
        </tr>
              <tr>
            <th>Priority</th>
            <td>High</td>
        </tr>
              <tr>
            <th>Sub-Tasks</th>
            <td>0/3 Completed</td>
        </tr>
              <tr>
            <th>Status</th>
            <td>In Development</td>
        </tr>
              <tr>
            <th>Assignee</th>
            <td>Julio Cesar de Oliveira Guimaraes</td>
        </tr>
              <tr>
            <th>Reporter</th>
            <td>Matt Hartel</td>
        </tr>
                </table>
          </details></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

33 files changed, 3749 insertions(+), 759 deletions(-)

This PR is too huge for one to review :broken_heart: 

Batman

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
Just starting reviewing :)

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/components/LicenseCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+type LicenseTiers = {
+	quantity?: number;
+	value?: number;
+};
+
+const LicenseSectorCard: React.FC<any> = ({
+	icon = <img alt="Info" src={infoCircleFullIcon} />,
+	licenseDescription,
+	licensetiers,
```
Comment (silasbsi) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/components/LicenseCard/index.tsx:22]: Please, change this name.
```suggestion
	licenseTiers,
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/components/LicenseSelector/index.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+					description="Try now. Pay Later"
+					disabled={!trialSKU}
+					icon={
+						<span className="license-icon">
+							<ClayIcon symbol="check-circle" />
+						</span>
+					}
+					onClick={
+						trialSKU ? () => setSelectedTimeline('trial') : () => {}
+					}
```
Comment (silasbsi) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/components/LicenseSelector/index.tsx:60]: Please, change this code.
```suggestion
					onClick={() => trialSKU ? setSelectedTimeline('trial') : {}}
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/components/PaidTimeline/index.scss
Comment (silasbsi) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/components/PaidTimeline/index.scss:1]: If the file does not have a function, remove it.

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions.

**Comment by kevenleone** [comment]
@juliocezar96 can you fix conflicts?

**Pull request closed without merging**